app/Lio/ServiceProviders/HashidsServiceProvider.php

```diff
@@ -5,15 +5,11 @@
 
 class HashidsServiceProvider extends ServiceProvider
 {
-    public function boot()
+    public function register() 
     {
-        $this->app->bind('Hashids\Hashids', function() {
+        $this->app->bind(['Hashids\Hashids' => 'hashids'], function() {
```

Comment:
Right. I thought you were mentioning the `register()` method rather than the callback. Will change that.